src/config.rs

```diff
@@ -465,9 +465,111 @@ pub struct DbOptions {
     ///
     /// Default: no TTL (insertions will remain until deleted)
     pub default_ttl: Option<u64>,
+
+    pub max_unflushed_memtable: usize,
 }
 
 impl DbOptions {
+    /// Converts the DbOptions to a JSON string representation
+    pub fn to_json_string(&self) -> Result<String, serde_json::Error> {
+        serde_json::to_string_pretty(self)
```

Comment:
Rather than pretty, can you do normal (ndjson)? This way it takes up a single line in the logs, which means it'll be more atomic in log serach.

Reply:
+1